app/controllers/reviews_controller.rb

```diff
@@ -1,11 +1,18 @@
 class ReviewsController < ApplicationController
+  skip_before_action :verify_authenticity_token
   def create
     @movie = Movie.find(review_params[:movie_id])
     @review = current_user.reviews.build(review_params)
```

Comment:
hạn chế sử dụng biến toàn cục em nhé, nếu ko dùng ở view thì để biến thường thôi